docs/guide/multisite.md

```diff
@@ -2,109 +2,189 @@
-                            Identifier\ContentType: [blog]
-                    blog_post:
-                        template: "full/blog_post.html.twig"
-                        match:
-                            Identifier\ContentType: [blog_post]
-        second_site:
-            pagelayout: second_site/pagelayout.html.twig
-            content_view:
-                full:
-                    blog_post:
-                        template: "second_site/full/blog_post.html.twig"
-                        match:
-                            Identifier\ContentType: [blog_post]
+        kids_group:
+            content:
+                tree_root:
+                    # LocationId of "Children"
+                    location_id: 57
 ```
 
-This config defines views that will be used for the `second_site` siteaccess: a separate `second_site/pagelayout.html.twig` and a template to be used for blog posts. When no view is defined under `second_site`, such as in the case of the `blog` Content Type, the template defined under `default` will apply. `default` will also be used for all siteaccesses other than `second_site`.
+Thanks to this config, you can access `<your site>/kids/Articles/Lions`, but not `<your site>/kids/General/Articles/Mammals`.
+
+In the next step, you need to reuse some content between sites, for example "Animal-photos" from "Media".
+You can allow the children's site to access them, even though they are in a different part of the tree, via `excluded_uri_prefixes`:
+
+``` yaml
+        kids_group:
+            content:
+                tree_root:
+                    location_id: 57
+                    excluded_uri_prefixes: [ /media/images/animal-photos ]
+```
 
-To load the base layout in templates you now need to use `{% extends noLayout == true ? viewbaseLayout : pagelayout %}`.
+At this point, using the `kids` SiteAccess, you gain access to `<your site>/kids/Media/Images/Animal-photos`,
+despite the fact that it is not a sub-item of the "Children" Location.
 
 ### Setting the Index Page
 
-The Index Page is the page shown when the root index / is accessed.
-You can configure the Index Page separately for each siteaccess. Put the parameter `index_page` in your `ezplatform.yml` file, under the right siteaccess category.
+The Index Page is the page shown when the root index `/` is accessed.
+You can configure the Index Page separately for each SiteAccess. Place the parameter `index_page` in your `ezplatform.yml` file, under the correct SiteAccess:
 
 ``` yaml
 # ezplatform.yml
 
 ezpublish:
     system:
-        mygreat_site:
-            languages:
-                - eng-US
-            #The page to show when accessing IndexPage (/)
-            index_page: /yourURIPage
+        kids_group:
+            # The page to show when accessing IndexPage (/)
+            index_page: /KidsFrontPage
 ```
 
 If not specified, the `index_page` is the configured content root.
+
+### Multisite with different repositories
```

Comment:
A bit unsure about the title.


A couple of things makes me unsure, as this page is about multi site in single database, and the limitation we are talking abut is in regards to multi repository setup. Do we have doc for the latter? Only thing I found was: https://doc.ezplatform.com/en/1.11/guide/repository/#multi-repository-setup


Cause if there where I would maybe clarify the relationship in the intro at the top of this page, and then here have a title ala `### Known limitations when using with Multi Repository setup`

Reply:
You are right, we lack a section on "multisite with multiple repositories". I'll add it to our to-do list, but I think this page would be the most logical place for it (then we can include this subsection into it).

Reply:
One lower hanging fruit for now could be to:
1. name this `### Limitations when using with Multi site URI matching with Multi Repository setup`
2. on first mention of multi repository use the link above even if it is super minimal on the concept :)